lib/bots/hackling.rb

```diff
@@ -0,0 +1,21 @@
+require "#{
```

Comment:
It's randomly selecting a bot to inherit from. To keep with the current pattern, I'm doing the random selection as part of the parameter passed to `require`, so I can require the right bot.